tests/integration/helpers/cluster.py

```diff
@@ -16,6 +16,12 @@
 import urllib.parse
 import shlex
 import urllib3
+from cassandra.policies import RoundRobinPolicy
+import cassandra.cluster
+import psycopg2
+import pymongo
```

Comment:
Are these changes relevant? Or it's just incorrectly resolved merge conflict?

Reply:
Changes in this pull request do not depend on these libraries.